### PR TITLE
[DOCS-6204] Add T-Engine to Outlook installation

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

File: microsoft-outlook/latest/install/index.md [hunk 1]
```diff
@@ -124,7 +122,76 @@ Ensure that you have applied the Alfresco Outlook Server AMP files ([see Install
...
+            - activemq
+    ```
+
+2. Add the following `JAVA_OPTS` variable to the `alfresco` container:
+
+    ```yaml
+    -DlocalTransform.transform-outlook.url=http://transform-outlook:8090/
+    ```
+
+3. **TODO** - needs more info: notes / start up / shutdown / troubleshooting...
```
Comment (mhofmann-itnovum) [inline microsoft-outlook/latest/install/index.md:192]: Link to https://github.com/Alfresco/acs-packaging/blob/master/docs/creating-a-t-engine.md

**Review by pnirwal — APPROVED** [review]

File: microsoft-outlook/latest/install/index.md [hunk 2]
```diff
@@ -124,7 +122,76 @@ Ensure that you have applied the Alfresco Outlook Server AMP files ([see Install
...
+### Install T-Engine on Tomcat
+
+If you wish to use an existing Tomcat application server, you can use the WAR bundle to install the Outlook Integration T-Engine.
+
+1. Install the latest version of [Tomcat 8.5](https://tomcat.apache.org/download-80.cgi){:target="_blank"} to run as a service.
+
+    See the [Tomcat documentation](https://tomcat.apache.org/tomcat-8.5-doc/index.html){:target="_blank"} for more information about configuring and using a Tomcat server.
+
+    For information about securing Tomcat, see [Tomcat security considerations](https://tomcat.apache.org/tomcat-8.5-doc/security-howto.html).
+
```
Comment (swall-developer) [inline microsoft-outlook/latest/install/index.md:171]: Add here as point 2: 
Rename war file from transform-outlook-webapp-${version}.war to transform-outlook.war

File: microsoft-outlook/latest/install/index.md [hunk 3]
```diff
@@ -124,7 +122,76 @@ Ensure that you have applied the Alfresco Outlook Server AMP files ([see Install
...
+2. Copy the WAR file into your `<TOMCAT_HOME>/webapps` folder.
+
+    The WAR (`transform-outlook-webapp-1.0.0.war`) is included in the distribution zip.
+
```
Comment (swall-developer) [inline microsoft-outlook/latest/install/index.md:175]: Add here:
To enable ActiveMQ in Outlook T-Engine set ActiveMQ url property in JAVA_OPTS variable:
-Dactivemq.url=${activemq.url}
To enable shared file store in Outlook T-Engine set shared file store url property in JAVA_OPTS variable:
-DfileStoreUrl=${shared.file.store.url}

**Review by pnirwal — APPROVED** [review]

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM.

File: microsoft-outlook/latest/install/index.md (hunk 3 above)
Comment (anxumalo) [reply microsoft-outlook/latest/install/index.md:183]: @swall-developer Thanks for the review and additional content - this has now been added in the latest commit (slightly reworded)

**Pull request merged**